WebHostLib/templates/playerOptions/playerOptions.html

```diff
@@ -0,0 +1,182 @@
+{% extends 'pageWrapper.html' %}
+{% import 'playerOptions/macros.html' as inputs %}
+
+{% block head %}
+    <title>{{ world_name }} Options</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename="styles/playerOptions/playerOptions.css") }}" />
+    <script type="application/ecmascript" src="{{ url_for('static', filename="assets/md5.min.js") }}"></script>
+    <script type="application/ecmascript" src="{{ url_for('static', filename="assets/js-yaml.min.js") }}"></script>
+    <script type="application/ecmascript" src="{{ url_for('static', filename="assets/playerOptions.js") }}"></script>
+
+    <noscript>
+        <style>
+            .js-required{
+                display: none;
+            }
+        </style>
+    </noscript>
+{% endblock %}
+
+{% block body %}
+    {% include 'header/'+theme+'Header.html' %}
+    <div id="player-options" class="markdown" data-game="{{ world_name }}" data-presets="{{ presets }}">
+        <noscript>
+            <div class="js-warning-banner">
+                This page has reduced functionality without JavaScript.
+            </div>
+        </noscript>
+
+        <div id="user-message">{{ message }}</div>
+
+        <div id="player-options-header">
+            <h1>{{ world_name }}</h1>
+            <h1>Player Options</h1>
+        </div>
+        <p>Choose the options you would like to play with! You may generate a single-player game from this page,
+            or download an options file you can use to participate in a MultiWorld.</p>
+
+        <p>
+            A more advanced options configuration for all games can be found on the
+            <a href="/weighted-options">Weighted options</a> page.
```

Comment:
```suggestion
            A more advanced options configuration for all games can be found on the
            <a href="/weighted-options">Weighted options</a> page.
```
This is outdated with the most recent changes. Currently the link just takes you back to the games page, but I think we should probably just reword it to link to the world's advanced options page.

Reply:
I fixed the link to point properly at the world's weighted options page.